strings/strings.go

```
@@ -343,3 +349,35 @@ func CommonSuffix(input ...string) string {
+			omission = opts.Omission
+		}
+	}
+
+	if len(input) <= length {
+		return input
+	}
+
+	// Consider omission length in the final string length
+	effectiveLength := length - len(omission)
+	if effectiveLength <= 0 {
+		effectiveLength = 1  // Ensure at least one character from input if possible
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)